strapi/strapi
# fix(admin): show purchase pages also with Cloud plans and change message

### What does it do?

It changes the purchase pages ids in the settings because these pages are shown also based on the permissions, and with the previous implementation these three pages were shown based on review-workflow, audit-logs and sso permissions, and it is not correct (because we want to show these pages when the feature is not allowed by the product license for marketing reasons). Previously for cloud licenses, we had these purchase pages hidden.
I changed the message on the body of these pages to add also Cloud licenses

### Why is it needed?

To show these purchase pages also with Cloud licenses

### How to test it?

Check the Settings pages with a Strapi cloud license, you need to see the 3 purchase pages with a lock icon, just ask me if you need to try it. 

### Related issue(s)/PR(s)

CS-638
